hybridse/src/udf/udf.cc

```diff
@@ -157,6 +157,16 @@ int32_t weekofyear(Date *date) {
     }
 }
 
+void int_to_char(int32_t val, StringRef* output) {
+    if (val > 256 || val < -256)
```

Comment:
what's the defined behavior if val < 0? Is there a clear definition for val < 0? @aceforeverd

Reply:
no clear definition. I think it should always be `val % 256`. 
This if check should removed

Reply:
Cool idea! I'll remove this check.